Example/Example/AppDelegate.swift

```diff
@@ -248,4 +248,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         
     }
     
+    func didUpdateClientLocationPermissionStatus(status: RadarLocationPermissionStatus){
```

Comment:
Why `didUpdateClientLocationPermissionStatus` instead of `didUpdateLocationPermissionStatus`? I feel like the latter is more concise. Is there any lack of clarity that removing `client` brings?

Reply:
Your right, I'll remove it